src/kubernetes/server/server-deployment.yaml

```diff
@@ -54,6 +54,8 @@ spec:
           value: 'true'
         - name: SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES_ENABLED
           value: 'true'
+        - name: SPRING_CLOUD_DATAFLOW_TASK_COMPOSED_TASK_RUNNER_URI
+          value: 'docker://springcloud/spring-cloud-dataflow-composed-task-runner:2.5.0.BUILD-SNAPSHOT'
```

Comment:
We would need to use the template files to update the version. Similar to what is done for the data flow server version update as in [here](https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/src/templates/kubernetes/server/server-deployment.yaml#L19)